godaddy-email-marketing.php

```diff
@@ -234,6 +234,26 @@ public function deactivate() {
 	 * Displays the admin notice.
 	 */
 	public function action_admin_notices() {
+
+		if ( get_option( 'wpem_gem_notice' ) ) {
+
+			?>
+
+			<div class="gem-notice notice notice-success is-dismissible">
+				<p>
+					<?php
+						printf(
+							__( 'Your website has a superpower: Email marketing. %1$s.', 'godaddy-email-marketing' ),
+							'<a href="' . admin_url( 'options-general.php?page=gem-settings' ) . '">' . __( 'Learn More', 'godaddy-email-marketing' ) . '</a>'
+						);
+					?>
+				</p>
+			</div>
+
+			<?php
+
```

Comment:
@EvanHerman I say we just delete the option right here after displaying the notice.